rtic-monotonics/src/nrf/rtc.rs

```diff
@@ -106,7 +106,10 @@ macro_rules! make_rtc {
 
                 $tq.initialize(Self {});
 
-                unsafe { pac::NVIC::unmask(Interrupt::$rtc) };
+                unsafe {
```

Comment:
Maybe a good time to add an (un)safety

```
# Safety
blabla
```
comment (here and for other similar ones)?